tests/inlineeditorui.js

```diff
@@ -100,6 +99,36 @@ describe( 'InlineEditorUI', () => {
 					positions: sinon.match.array
 				} );
 			} );
+
+			it( 'pin() is not called on editor.ui#update when panel is hidden', () => {
+				const spy = sinon.stub( view.panel, 'pin' );
+
+				view.panel.hide();
+
+				editor.ui.fire( 'update' );
+				sinon.assert.notCalled( spy );
+			} );
+
+			// Sometimes this test can fail, due to the fact that it have to wait for async ResizeObserver execution.
+			it( 'should set inline toolbar max-width to the width of the editable element', done => {
```

Comment:
If the resizer logic goes from `InlineEditorUI` to `InlineEditorUIView`, the tests must follow.

Reply:
I moved it too and set `shouldGroupWhenFull` explicitly before test, to make sure it's not `undefined` (in the case when `editable` and `options` are not passed to the `InlineEditorUIView`)